### ISSUE TITLE
check for cvmfs before starting condor

### ISSUE THREAD

**Issue by dsschult** [WIPACrepo/pyglidein#64]
We should check for cvmfs before starting condor, since it's basically required for our jobs.

Simple check is:
```bash
if [ -e /cvmfs/icecube.opensciencegrid.org/py2-v1/setup.sh ]; then
    CVMFS="true"
fi
```

Harder is to launch parrot in the else case, and check again.

**Comment by briedel** [comment]
Would also help with moving condor cvmfs

**Comment by gonzalomerino** [comment]
Hi David,

OSG glideinWMS glideins publish this classad after checking for our repo

HAS_CVMFS_icecube_opensciencegrid_org

true, or false


could we adopt the same one in pyglidein?

Gonzalo

**Comment by dsschult** [comment]
Gonzalo:  already done in df26634
